playbooks/roles/vault/tasks/main.yml

```diff
@@ -61,7 +61,7 @@
 
 - name: Install Vault environment variables
   lineinfile:
-    path: "/home/{{ ansible_ssh_user }}/.bashrc"
+    path: "{{ ansible_env.HOME }}/.bashrc"
```

Comment:
Note that `~` or `$HOME` could be used instead of `ansible_env.HOME`, because [type of `path` parameter is `path`](https://github.com/ansible/ansible/blob/3408de156ab089e9f05c912d52869fee20fa5fc8/lib/ansible/modules/files/lineinfile.py#L34) and [`path` type parameters automatically expand `~`/`$HOME`](https://github.com/ansible/ansible/blob/3408de156ab089e9f05c912d52869fee20fa5fc8/lib/ansible/module_utils/basic.py#L1949-L1951).

Reply:
Excellent, it's less arcane :+1: